mycosnp_results_files.py

```diff
@@ -0,0 +1,256 @@
+arln_labware_cols  = ['LIMS_TEST_ID', 'Passed QC', 'ARLN WGS ID', 'GC WGS ID', 'WGS Organism', 'WGS MLST', 'Run Name', 'Genomic Coverage', 'AMR Gene symbol', 'AMR Name of closest sequence', 'AMR % Coverage of reference sequence', 'AMR % Identity to reference sequence', 'Virulence Gene symbol', 'Virulence Name of closest sequence', 'Virulence % Coverage of reference sequence', 'Virulence % Identity to reference sequence', 'SRA Accession', 'BioSample Accession', 'Genbank Accession', 'GC Percent', 'Avg PHRED', 'Clade']
+
+# creating columns for unused values 'SRA Accession', 'BioSample Accession', 'Genbank Accession', 'GC WGS ID', 'Clade'
+optional_columns = [
+    'ARLN WGS ID', 
+    'SRA Accession', 
+    'BioSample Accession', 
+    'Genbank Accession', 
+    'GC WGS ID',  
+    'Clade',
+    'WGS MLST', 
+    'AMR Gene symbol', 
+    'AMR Name of closest sequence', 
+    'AMR % Coverage of reference sequence', 
+    'AMR % Identity to reference sequence', 
+    'Virulence Gene symbol',
+    'Virulence Name of closest sequence',
+    'Virulence % Coverage of reference sequence', 
+    'Virulence % Identity to reference sequence',
+    'Subtype', 
+    'Date Uploaded to CDC', 
+    'Date Uploaded to NCBI/SRA' ]
+
+for unused in optional_columns:
+    if unused not in df.columns:
+        df[unused] = ''
+
+# Get final columns 'LIMS_TEST_ID', 'Passed QC', 'WGS Organism', 'Run Name', 'Genomic Coverage', 'GC Percent', 'Avg PHRED'
+df['LIMS_TEST_ID']                  = df['lims_id']
+df['Passed QC']                     = df['Pass']
+df['WGS Organism']                  = df['organism']
+df['Run Name']                      = df['Sample_Project']
+df['Genomic Coverage']              = df['Reference Length Coverage After Trimming']
+df['GC Percent']                    = df['GC After Trimming']
+df['Avg PHRED']                     = df['Average Q Score After Trimming']
+
+# Create subset of samples for Labware 8 import files
+sample_list = []
+
+if args.list and os.path.exists(args.list) :
+    print("Creating files for subset of samples")
+    with open(args.list) as f:
+        sample_list = f.read().splitlines()
+else:
+    print("Creating Labware 8 files for each sample")
+    sample_list = df[df['Pass'] == 'Y']['LIMS_TEST_ID'].to_list()
+
+if not os.path.exists("labware8"):
+    os.mkdir("labware8")
+
+for sample in sample_list:
+    sample_df       = df[df['LIMS_TEST_ID'] == sample ].copy()
+    sample_name     = str(sample_df['Sample_ID'].values[0])
+    sample_organism = str(sample_df['WGS Organism'].values[0])
+
+    # Writing the files for each sample separately
+    arln_filename = 'labware8/arln_' + sample + '_results.txt'
+    sample_df.to_csv(arln_filename, columns=arln_labware_cols, index=False, sep='\t')
```

Comment:
The file needs to be a csv file, so please delete the '\t'

Reply:
changed  the sep='\t' to sep=',' and changed 'results.txt' to 'results.csv'